### ISSUE TITLE
Customizations lost!

### ISSUE THREAD

**Issue by MitchWagnerSD** [scripting/feedlandSupport#242]
I signed in to Feedland and saw that I already had an account there with a bunch of feeds. I have tried feedland before. I deleted all those and re-added a few by hand, then imported some more from an OPML file I downloaded from Inoreader, my previous newsreader. Then I added a category and manually added a dozen or more feeds to that category, which was somewhat time-consuming because I had to add each one individually. 

Then I was done and I looked at the result and said, "Very nice!"

Then I switched to another browser and logged in there. And I found all my customizations were lost. I was looking at the same set-up I started with an hour or two ago. 

Super-frustrating!

**Comment by scripting** [comment]
Sorry to hear that this happened. It's hard to know what went wrong without some basic info about what happened, what kinds of customizations you did, maybe there was a misunderstanding about how something works.

**Comment by MitchWagnerSD** [comment]
Thanks, Dave. While I appreciate the opportunity to test Feedland, I think I'll stick with Inoreader, at least for now. 

I was able to restore my customizations—for some reason, the changes were still current in the first browser I used, so I exported the OML, and then imported them into Feedland using the second browser I was using.

I created one category, called "front page,"  and found when I viewed the "latest news" link on that category, I saw the latest news from all my feeds. This was not the behavior I expected; I expected to only see the latest news from that particular category.
